test/Microsoft.Extensions.Configuration.Xml.Test/XmlConfigurationTest.cs

```diff
@@ -300,8 +300,7 @@ public void ThrowExceptionWhenFindDTD()
                     </Data>
                 </settings>";
             var xmlConfigSrc = new XmlConfigurationProvider(ArbitraryFilePath);
-            var isMono = Type.GetType("Mono.Runtime") != null;
-            var expectedMsg = isMono ? "Document Type Declaration (DTD) is prohibited in this XML.  Line 1, position 10." : "For security reasons DTD is prohibited in this XML document. "
+            var expectedMsg = "For security reasons DTD is prohibited in this XML document. "
```

Comment:
This still seems to be failing on Travis. I think verifying that the message contains words "DTD", "prohibited" "Xml" should be enough.